frontend/apps/filemanager/filemanagerutil.lua

```diff
@@ -29,12 +29,36 @@ function filemanagerutil.abbreviate(path)
     return path
 end
 
--- Purge doc settings in sidecar directory,
+-- Purge doc settings in sidecar directory
 function filemanagerutil.purgeSettings(file)
     local file_abs_path = util.realpath(file)
     if file_abs_path then
         DocSettings:open(file_abs_path):purge()
     end
 end
 
+-- Purge doc settings except kept
+function filemanagerutil.purgeViewSettings(file)
```

Comment:
Not sure it is now appropriate to have this defined in filenamanagerutil - as we don't invoke it at all from FileManager - and it is quite not different than the above function.
But I can't think of an obvious better place for this (docsettings.lua is a generic and currently doesn't know about any real setting).

Reply:
I guess this function name can now be `filemanagerutil.resetDocumentSettings(file)`